.github/workflows/deploy-nuxeo.com-benchmarks.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Deploy Nuxeo Benchmarks
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Install Hugo
+      run: |
+        wget https://github.com/spf13/hugo/releases/download/v0.80.0/hugo_0.80.0_Linux-64bit.tar.gz -O /tmp/hugo.tgz
+        (cd /tmp/; tar xzvf hugo.tgz)
+
+    - name: Build Hugo Site
+      run: /tmp/hugo -t hyde
```

Comment:
You might want to take alook at https://github.com/marketplace/actions/hugo-site that builds the site without installation requirements.

Reply:
don't like it?